terraform/aws.tf

```diff
@@ -0,0 +1,30 @@
+terraform {
+    backend "s3" {
```

Comment:
I didn't knew it's stateful. Too bad. What happens if someone deletes the state?

Reply:
we have issue, i will setup this bucket to be versioned ;]